multiplex.py

```diff
@@ -258,6 +261,54 @@ def convert_vals(obj):
 
     return new_obj
 
+def load_presets(json_req):
+    """Create a dict for essentials, defaults and includable presets"""
+    if bool(json_req) and "presets" in json_req:
+        # copy the essentials and defaults to a dict
+        # to make our lives easier when overriding params
+        if "essentials" in json_req["presets"]:
+            for _ess in json_req["presets"]["essentials"]:
+                essentials_dict.update({ _ess["arg"]: _ess["vals"] })
+        if "defaults" in json_req["presets"]:
+            for _def in json_req["presets"]["defaults"]:
+                defaults_dict.update({ _def["arg"]: _def["vals"] })
```

Comment:
Is there a reason why you copy these individual parameters one at a time rather than copying the whole set at once?

Reply:
Yes, logic simplification. 
All the params are copied into a simpler dict to ease search and override logic.
```
        "essentials": [
            { "arg": "bs", "vals": ["1K"] },
            { "arg": "duration", "vals": ["60"] }
        ],
```
becomes...
```
{ "bs": ["1k"] , "duration": ["60"] }
```
Instead of iterating the big structure and looking at the content of arg, we simply do:
```if param in my_simple_dict:```